client/src/featureform/resources.py

```diff
@@ -650,12 +650,12 @@ def _create_training_set_resource(self, db) -> None:
 
     def _check_insert_training_set_resources(self, db) -> None:
         try:
-            db.getNameVariant("labels_variant", "labelName", self.label[0], "variantName", self.label[1])
+            db.get_name_variant("labels_variant", "labelName", self.label[0], "variantName", self.label[1])
```

Comment:
Has this branch been rebased with the magic numbers one?

Reply:
Yes it has